### PR TITLE
add wait time for 9552

### PR DESCRIPTION
**Pull request by zhouying7780**

### CODE REVIEW

**Comment by zhouying7780** [comment]
http://pastebin.test.redhat.com/1078419

@xingxingxia Please help review , thanks

File: features/cli/policy.feature [hunk 1]
```diff
@@ -476,19 +476,25 @@ Feature: change the policy of user/service account
...
+    Then I wait for the steps to pass:
+    """
     Given I run the :policy_scc_review client command with:
       | serviceaccount | default                      |
       | f              | PodSecurityPolicyReview.json |
       | n              | <%= project.name %>          |
     Then the step should succeed
     And the output should not match:
       | .*default.*restricted |
+    """
```
Comment (xingxingxia) [inline features/cli/policy.feature:497]: For "the output should **not** match", waiting is not needed. Adding the waiting time will be unnecessarily confusing. Could you remove?

**Comment by zhouying7780** [comment]
updated , please help review again, thanks

**Comment by xingxingxia** [comment]
/lgtm
@liangxia help merge
Thanks!

**Pull request merged**